src/fragments/gen2/quickstart/build-a-backend.mdx

```diff
@@ -4,11 +4,11 @@ Next, we will add data and auth capabilities to the app.  In Amplify (Gen 2), th
 
 ### Add data
 
-`create-amplify` provides the scaffolding of a `/amplify/data/resource.ts` which is ready to deploy.
+`create-amplify` provides the scaffolding of a `/amplify/data/resource.ts` file, which is ready to deploy.
 
 <Accordion title="See the complete /amplify/data/resources.ts">
 
-If you open up the `/amplify/data/resource.ts` file in your text editor, you will see some generated code already there.
+If you open the `/amplify/data/resource.ts` file in your text editor, you will see some generated code already there.
```

Comment:
A suggestion to make this a statement instead of starting with a question "If you..." and update the language to accurately describe what's in the file

```suggestion
Open the `/amplify/data/resource.ts` file in your text editor, and you will see a default data model generated for you.
```